stable/presto/templates/coordinator-configmap.yaml

```diff
@@ -73,9 +73,26 @@ data:
 {{ include "v3io-configs.script.httpHealthCheckWithJava" . | indent 4 }}
 
   docker-presto.sh: |
-    #!/bin/bash 
+    #!/bin/bash
     cp {{ .Values.server.config.mapPath }}/*.properties $PRESTO_CONF_DIR
     cp {{ .Values.server.config.mapPath }}/*.config $PRESTO_CONF_DIR
+
+{{- if .Values.hive.metastore }}
+    # Hive configuration
+    mv $PRESTO_CONF_DIR/hive.properties $PRESTO_CONF_DIR/catalog/hive.properties
+
+    # Hive dependencies
+    ln -s /presto/plugin/v3io/v3io-hcfs_2.11.jar /presto/plugin/hive-hadoop2/v3io-hcfs_2.11.jar
```

Comment:
instead of `2.11` use `$SPARK_MAJOR_VERSION`

Reply:
you probably mean `SCALA_MAJOR_VERSION`